include/simdjson/dom/element-inl.h

```diff
@@ -375,6 +375,23 @@ inline simdjson_result<element> element::operator[](const char *key) const noexc
   return at_key(key);
 }
 
+inline bool is_pointer_well_formed(std::string_view json_pointer) noexcept {
+  if (simdjson_unlikely(json_pointer[0] != '/')) {
```

Comment:
I understand here that the caller is responsible to ensure that `std::string_view` is not empty. **You do not need to** but you could add a remark to that effect (so that people do not think that the code is buggy).

Reply:
This function is very close to the caller piece, so I think it's ok. And since it should be called only at this place, I don't think it would be confusing.